deploy/examples/create-external-cluster-resources.py

```diff
@@ -50,22 +47,6 @@
     print(f"Error: {noModErr}\nExiting the script...")
     sys.exit(1)
 
-try:
-    # for 2.7.x
-    from StringIO import StringIO
-except ModuleNotFoundError:
-    # for 3.x
-    from io import StringIO
-
-try:
-    # for 2.7.x
-    from urlparse import urlparse
-    from urllib import urlencode as urlencode
-except ModuleNotFoundError:
-    # for 3.x
-    from urllib.parse import urlparse
-    from urllib.parse import urlencode as urlencode
-
 try:
```

Comment:
This could also be dropped since encodestring is deprecated since 3.1?

Reply:
updated